lib/install/management/virtual_app.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/vmware/vic/lib/install/data"
 	"github.com/vmware/vic/pkg/errors"
 	"github.com/vmware/vic/pkg/trace"
+  "github.com/vmware/vic/pkg/version"
```

Comment:
The [spacemacs go layer](https://github.com/syl20bnr/spacemacs/tree/master/layers/%2Blang/go) runs `go fmt` on every file save.